src/OptimalTransport.jl

```diff
@@ -192,7 +199,11 @@ function sinkhorn(mu, nu, C, eps; kwargs...)
     # compute dual potentials
     u, v = sinkhorn_gibbs(mu, nu, K; kwargs...)
 
-    return Diagonal(u) * K * Diagonal(v)
+    if mu isa Vector
+        return Diagonal(reshape(u, :)) * K * Diagonal(reshape(v, :))
+    else
+        return [Diagonal(u[:, i]) * K * Diagonal(v[:, i]) for i in 1:size(mu, 2)]
```

Comment:
It seems this assumes that `u` and `v` contain the same number of histograms. I don't think this is checked anywhere?